### PR TITLE
Add create ainft server app script

### PR DESCRIPTION
**Pull request by woomurf**
- Add create ainft server app script (from [ainft-js](https://github.com/ainft-team/ainft-js/tree/main/tools/create-ainft-server-app))

### CODE REVIEW

File: tools/create-ainft-server-app.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  console.log(
+    '<APP ID>: This is the app id to create on the ainft server.' +
+      'Combinations of lowercase letters, underscores and number are allowed.'
+  );
+  console.log(
+    '<USER ID>: This is the user id used in the ainft server. Become the owner of the app.' +
+      'It is recommended to use the userId of the place where ainft-js is used.'
+  );
+  console.log(
+    '<PRIVATE_KEY> (optional): It means ain blockchain private key. If not entered, it will be created automatically.\n'
```
Comment (platfowner) [inline tools/create-ainft-server-app.js]: What about adding some comments for the created private key? e.g.
"The created private key should be kept safe to be used for accessing AINFT Server APIs with the app id."

Comment (platfowner) [inline tools/create-ainft-server-app.js]: Are we providing a guide for the private key creation? e.g. providing a separate script for creating a private key?
or a guide to https://github.com/ainblockchain/ain-blockchain/blob/develop/tools/account/createAccounts.js ?
Would it be helpful to provide a similar script in this repo?

Comment (woomurf) [reply tools/create-ainft-server-app.js]: Done!

Comment (woomurf) [reply tools/create-ainft-server-app.js]: Done. Added createAccessKey.js script and change variable name from privateKey to AccessKey.

**Comment by woomurf** [comment]
@platfowner PTAL!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**